exericse4/exercise2.js

```diff
@@ -0,0 +1,10 @@
+function isPrimeNumber(number) {
+  if (number < 2) return false;
+  if (number === 2) return true;
+  for (let i = 2; i < number; i++) {
+    if (number % i === 0) return false;
+    return true;
```

Comment:
return true; must go out of the loop because it is going to make loop break without checking for many numbers, for example 15 is returning true yet is not true